includes/commerce_license_billing.charge.inc

```diff
@@ -0,0 +1,82 @@
+<?php
+
+interface CommerceLicenseBillingChargeInterface extends ArrayAccess {
+  public function match($line_item_wrapper);
+}
+
+class CommerceLicenseBillingCharge implements CommerceLicenseBillingChargeInterface {
+
+    const SCHEMA = [
```

Comment:
An array in a constant is a PHP 5.6+ thing only.

Why not use actual class properties?